### ISSUE TITLE
Loads of JSON errors using Feedly

### ISSUE THREAD

**Issue by hadess** [jangernert/FeedReader#894]
With FeedReader from Flathub:
```sh
$ flatpak info org.gnome.FeedReader

FeedReader - RSS client for various webservices

          ID: org.gnome.FeedReader
         Ref: app/org.gnome.FeedReader/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 2.8.2
     License: GPL-3.0+
      Origin: org.gnome.FeedReader-origin
  Collection: org.flathub.Stable
Installation: system
   Installed: 10.3 MB
     Runtime: org.gnome.Platform/x86_64/3.30
         Sdk: org.gnome.Sdk/x86_64/3.30

      Commit: 28030505a98e6176b2928eab5b3812e842acbfc6d1f4a059439c17899a99e813
      Parent: e3f7441cd574fc9edbefc787b2c9be952357ad595ceaae62caaebb09f58b589f
     Subject: v2.8.2 (9922929d)
        Date: 2019-02-11 01:10:10 +0000
```

I still see the same errors using Feedly as I saw in #509:
```
$ journalctl -b | grep 'json_object_get_' | grep 'node != NULL' | wc -l
1000146
$ uptime
 01:57:42 up 11 days, 13:34,  1 user,  load average: 0.80, 1.29, 1.01
$ journalctl -b | grep 'json_object_get_' | grep 'node != NULL' | cut -d ' ' -f 10- | sort | uniq
json_object_get_boolean_member: assertion 'node != NULL' failed
json_object_get_string_member: assertion 'node != NULL' failed
```

The errors take the form:
```
classic org.gnome.FeedReader.desktop[1900]: (feedreader:2): Json-CRITICAL **: 06:04:19.118: json_object_get_string_member: assertion 'node != NULL' failed
classic org.gnome.FeedReader.desktop[1900]: (feedreader:2): Json-CRITICAL **: 06:04:19.119: json_object_get_boolean_member: assertion 'node != NULL' failed
```

**Comment by brendanlong** [comment]
I think in normal usage these warnings shouldn't matter, but I created #905 to get rid of these

**Comment by brendanlong** [comment]
This will be fixed in v2.9.0
